upcoming_match.py

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env python
```

Comment:
This file has some small utility as an example or if you don't have/get scriptine. I could go either way on whether to keep it.